BLSQ/iaso
# IA-2960: Group permissions under clear titles

Explain what problem this PR is resolving
- Group permissions
Related JIRA tickets : [IA-2960](https://bluesquare.atlassian.net/browse/IA-2960)

## Self proofreading checklist


## Doc
- **iaso/docs/pages/dev/how_to/add_new_permission/add_new_permission.md**

## Changes
- Created PERMISSIONS_PRESENTATION constant
- Created a new api for permissions

## How to test
- Go in user and user role menu
- Edit or create a user and user role
- Check if permissions are grouped

## Print screen / video
[Screencast from 2024-05-22 15-30-18.webm](https://github.com/BLSQ/iaso/assets/19631540/d7fd509b-f43d-4a2d-99e0-7ab250b1a2a8)




[IA-2960]: https://bluesquare.atlassian.net/browse/IA-2960?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
